ui/job-view/pushes/PushActionMenu.jsx

```diff
@@ -126,12 +130,12 @@ export default class PushActionMenu extends React.PureComponent {
             <li
               title="Hide Runnable Jobs"
               className="dropdown-item"
-              onClick={() => hideRunnableJobsCb()}
+              onClick={() => hideRunnableJobs()}
```

Comment:
I think you can just pass in `hideRunnableJobs` here and on line 138rather than calling/returning it with the arrow function.

Reply:
Ahh, good catch.  Fixed.  :)